store/ARC2_StoreDeleteQueryHandler.php

```diff
@@ -46,9 +46,13 @@ public function runQuery($infos)
         if ($tc && ($this->refs_deleted || (1 == rand(1, 100)))) {
             $this->cleanTableReferences();
         }
+        // TODO What does this rand() call here? remove it and think about a cleaner way
```

Comment:
Yeah, not very clean. It was just based on experience that MyISAM breaks on a regular basis, and repairing the tables once in 100 queries, and cleaning up unused values every 500 queries was a pragmatic hack to keep the store in a working state.